packages/base/src/SpineBase.ts

```diff
@@ -98,10 +104,12 @@ export abstract class SpineBase<Skeleton extends ISkeleton,
         this._debug = value;
     }
 
+    private SpriteCtor: SpineSpriteConstructor;
+    private MeshCtor: SpineMeshConstructor;
 
     abstract createSkeleton(spineData: ISkeletonData);
 
-    constructor(spineData: SkeletonData) {
+    constructor(spineData: SkeletonData, SpriteCtor?: SpineSpriteConstructor, MeshCtor?: SpineMeshConstructor) {
```

Comment:
please make second params an object

Reply:
in case we'll need custom graphics for masks, e.t.c. ) Better to store those things in a object as second param